migrations/021.do.sucessful-http-retrieval-stats.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE retrieval_stats
+ADD COLUMN IF NOT EXISTS successful_http INT NOT NULL DEFAULT 0
```

Comment:
I am wondering if setting `successful_http = 0` is the right approach for historical data. The real RSR was almost certainly non-zero, probably in the range of ~5%.

How about making this column nullable?

```suggestion
ADD COLUMN IF NOT EXISTS successful_http INT
```

This way, spark-stats REST API handler can exclude HTTP-RSR for days where we did not capture it. As a result, historical charts will not show any HTTP-RSR value for those days. (In your current proposal, historical charts would show 0%.)

Reply:
Good idea, I made the change.